server/server.go

```diff
@@ -455,13 +455,14 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		return nil
 	}
 
-	msg := fmt.Sprintf("Tag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+	username := slashCommand.Username
+	msg := fmt.Sprintf("Username: `@%s`\nTag %s created by. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", username, tag)
```

Comment:
Could the text be more self explaining? From reading the plain text it it's obvious to me, what `Username` refers to. Maybe something like: `@abc triggered a plugin release process.\n[Rest of the Text]`

Reply:
Yeah fair point. 👍